staging/src/k8s.io/kubectl/docs/book/pages/extending_kubectl/discovering_plugins.md

```diff
@@ -21,7 +21,7 @@ Krew helps to
 Krew should be used as a kubectl plugin. To set yourself up to using krew, you need to do two things:
 
 1. Install git
-1. Install krew as described on the project page [krew.dev](https://github.com/kubernetes-sigs/krew/#installation).
+1. Install krew as described on the project page [sigs.k8s.io/krew](https://sigs.k8s.io/krew/#installation).
```

Comment:
Ditto we probably need to de-dupe all these links into one as I'm moving install docs to the website.